src/internal/components/pluggableVisualizations/headline/PluggableHeadline.tsx

```diff
@@ -221,4 +225,15 @@ export class PluggableHeadline extends AbstractPluggableVisualization {
             return resultItems;
         }, []);
     }
+
+    private setConfigFromFeatureFlags(config: IChartConfig): IChartConfig {
```

Comment:
I just checked the PR out of curiosity and this block of code is repeated twice.  
Also, the second `if` is unnecessary.

`if (this.featureFlags && this.featureFlags.disableKpiDashboardHeadlineUnderline === true)`

Maybe I would one lined it as: 

````
return this.featureFlags && this.featureFlags.disableKpiDashboardHeadlineUnderline === true
    ? { ...config, disableDrillUnderline: true }
    : config;
````

(I don't think that functions that modify its parameters are good practice)

Reply:
Thx for comment. Fixed

Reply:
You still have it here twice ;) imho
src/components/uri/Visualization.tsx:369

Reply:
Ok I will move this method to service